calico/metadata.csv

```diff
@@ -0,0 +1,19 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+calico.felix_active_local_endpoints,gauge,,,,Number of active endpoints on this host,0,calico,calico.felix_active_local_endpoints
```

Comment:
this may need to be updated if you choose to use the pseudo-hierarchical metric name suggestions